sessions/css-flexbox/profile/css/styles.css

```diff
@@ -0,0 +1,53 @@
+/* Scroll down to get to the exercise! */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	line-height: 1.5;
+	padding: 20px;
+}
+
+h1,
+h2 {
+	line-height: 1.2;
+	border-bottom: 5px dotted indigo;
+	text-align: center;
+}
+
+img {
+	border-radius: 50%;
+	max-width: 50%;
+	height: auto;
+}
+
+/* ------- The exercise starts here ------- */
+
+/* Exercise 1: The first section contains a image and a about text. Style this section with flexbox so that the image is centered.*/
```

Comment:
```suggestion
/* Exercise 1: The first section contains an image and an about text. Style this section with flexbox so that the image is centered.*/
```